src/design-tokens/sizes.yml

```diff
@@ -1,49 +1,53 @@
 props:
-  edge_small:
+  - name: edge_small
     value: 0.0625rem
     type: relative
     category: sizing
-  edge_medium:
+  - name: edge_medium
     value: 0.125rem
     type: relative
     category: sizing
-  edge_large:
+  - name: edge_large
     value: 0.25rem
     type: relative
     category: sizing
-  radius_medium:
+  - name: radius_medium
     value: 0.25em
     type: relative
     category: radius
-  control_height:
+  - name: radius_full
+    value: 9999px
```

Comment:
For my own knowledge gathering, can you comment on why `9999px` was preferred here, instead of a high percentage based value?

Reply:
Great question, I will add a `comment` value (and a new issue to _display_ comments when they exist in our design token docs)

Reply:
✅ Comment added